listenbrainz/db/user.py

```diff
@@ -117,7 +118,7 @@ def get_by_token(token):
     """
     with db.engine.connect() as connection:
         result = connection.execute(sqlalchemy.text("""
-            SELECT id, created, musicbrainz_id
+            SELECT id, created, musicbrainz_id, last_login
```

Comment:
I guess that makes sense for all the `get_by_*` functions right now. There may be other functions that don't need all the user select fields later.